src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -14,4 +14,5 @@ public class CliSyntax {
     public static final Prefix PREFIX_KEY = new Prefix("k/");
     public static final Prefix PREFIX_POSTALCODE = new Prefix("c/");
     public static final Prefix PREFIX_UNITNUMBER = new Prefix("u/");
+    public static final Prefix PREFIX_TYPE = new Prefix("y/");
```

Comment:
is this a typo haha? did u mean to use 't'? I thought it could be a typo since 'y' is just beside 't'

Reply:
Not a typo. the t is already used by `PREFIX_TAG`

Reply:
fixed